docs/docs/reference/release-notes/v2-maintenance-plan.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Gatsby Framework Version Support
+---
+
+## Overview
```

Comment:
nit: swapping this out to "Introduction" to match the template (https://www.gatsbyjs.com/contributing/docs-contributions/how-to-write-a-reference-guide/)

```suggestion
## Introduction
```

Reply:
Make sense! Consistency is key!